community/installation-guides/panel/debian11.md

```diff
@@ -0,0 +1,65 @@
+# Debian 11
+In this guide we will install Pterodactyl v1.X — including all of it's dependencies
+
+[[toc]]
+
+::: tip
+This guide is based off the [official installation documentation](/panel/1.0/getting_started.md) but is tailored specifically for Debian 11.
+:::
+
+## Install Requirements
+```bash
+apt -y install software-properties-common curl ca-certificates gnupg lsb-release sudo wget
+```
+
+### php 8.1
+```bash
+# Add repository for PHP
+echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/sury-php.list
+
+wget -qO - https://packages.sury.org/php/apt.gpg | sudo apt-key add -
```

Comment:
Adding `wget` package into the mix is unnecessary. Curl is used everywhere and most of the time preinstalled, so please use it. gnupg2 is generally used as well instead of the classic one.

`apt-key `is also deprecated. The signing key can be added and converted from PGP into GPG with `sudo gpg --dearmor -o <keyring file path>`

Should I remember Debian paths right it would be:

`curl -fsSL  https://packages.sury.org/php/apt.gpg| sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/sury-keyring.gpg`